docs/content/s3.md

```diff
@@ -6040,3 +6041,121 @@ d) Delete this remote
 
 y/e/d> y
 ```
+### Zata Object Storage {#Zata}
+
+[Zata Object Storage](https://zata.ai/) provides a secure, S3-compatible cloud storage solution designed for scalability and performance, ideal for a variety of data storage needs.
+
+Run:
+go build -o rclone.exe
+
+The S3 compatible gateway is configured using `rclone.exe config` with a type of `s3` and a provider name of `Zata`. Below is an example run of the configurator.
+
+First run:
+
+```
+rclone.exe config
+```
+
+```
+This will guide you through an interactive setup process:
+No remotes found, make a new one?
+n) New remote
+s) Set configuration password
+q) Quit config
+n/s/q> n
+
+Enter name for new remote.
+name>My zata storage
+
+Type of storage to configure.
+Choose a number from below, or type in your own value.
+XX / Amazon S3 Compliant Storage Providers including AWS,(included Zata also) ...
+\ (s3)
+Storage> 4 (Press with the matching number) Or s3
+
+Choose your S3 provider.
+Choose a number from below, or type in your own value.
+36 / Zata (S3 compatible Gateway)
+\ (Zata)
+provider> Zata
+
+Get AWS credentials from runtime (environment variables or EC2/ECS meta data if no env vars).
+Only applies if access_key_id and secret_access_key is blank.
+Choose a number from below, or type in your own boolean value (true or false).
+Press Enter for the default (false).
+1 / Enter AWS credentials in the next step.
+\ (false)
+2 / Get AWS credentials from the environment (env vars or IAM).
+\ (true)
+env_auth> 1
+
+AWS Access Key ID.
+Leave blank for anonymous access or runtime credentials.
+Enter a string value. Press Enter for the default ("").
+access_key_id> "Your access Key ID"
+
+AWS Secret Access Key (password).
+Leave blank for anonymous access or runtime credentials.
+Enter a string value. Press Enter for the default ("").
+secret_access_key> "Your secret access Key ID"
+
+Region where you can connect with.
+Choose a number from below, or type in your own value.
+Press Enter to leave empty.
+1 / Indore, Madhya Pradesh, India
+\ (us-east-1)
+region> 1
+
+Endpoint for Zata Object Storage.
+Choose a number from below, or type in your own value.
+Press Enter to leave empty.
+1 / South Asia Endpoint
+\ (idr01.zata.ai)
+endpoint> 1
+
+Endpoint for S3 API.
```

Comment:
Why is it asking for Endpoint twice? If it is then you've made a mistake with the `Provider` lines in the config. You need to exclude this question for `Zata`.